tests/conftest.py

```diff
@@ -0,0 +1,9 @@
+
+import pytest
+
+from time_tracker_api import create_app
+
+@pytest.fixture(scope='session')
+def app():
+    app = create_app()
+    return app
```

Comment:
`return create_app()`
it's just a suggestion

Reply:
Makes sense, I just know that here in the middle I will have to do adjustments.

Reply:
Ah OK, I understand :+1:

Reply:
But I applied your suggestion tho 👍